src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceOutputFormatProvider.java

```diff
@@ -51,37 +51,38 @@ public SalesforceOutputFormatProvider(SalesforceSinkConfig config) {
       .put(SalesforceSinkConstants.CONFIG_ERROR_HANDLING, config.getErrorHandling().getValue())
       .put(SalesforceSinkConstants.CONFIG_MAX_BYTES_PER_BATCH, config.getMaxBytesPerBatch().toString())
       .put(SalesforceSinkConstants.CONFIG_MAX_RECORDS_PER_BATCH, config.getMaxRecordsPerBatch().toString());
+    if (config.getConnection() != null) {
```

Comment:
config.connection will not be null in prepareRun() so the null check is not required here

Reply:
Removed